config.json

```diff
@@ -14,6 +14,16 @@
         "text formatting"
       ]
     },
+	{
+	  "slug": "isogram",
+	  "difficulty": 1,
+	  "topics": [
+	    "arrays",
+	    "strings",
+		"control-flow (if-else statements)",
+		"control-flow (loops)"
```

Comment:
Ha ha, that article is getting huge coverage! Normally I use spaces but did this on an old machine I had and was fighting Notepad++ on the tabs/spaces front 😕